packages/core/src/internal/utils/metadata.ts

```diff
@@ -18,7 +18,7 @@ export function getElementStorybookArgs(args: any) {
 
 function getCustomPropsStorybookArgs(customElementMetadata: any) {
   return customElementMetadata.cssProperties?.reduce((obj: any, prop: { name: string }) => {
-    const type = prop?.name?.includes('color') || prop.name === '--background' ? 'color' : 'text';
+    const type = prop?.name?.includes('color') || prop.name?.endsWith('background') ? 'color' : 'text';
```

Comment:
wouldn't we need this to be...

```
const type = prop?.name?.includes('color') || prop?.name?.endsWith('background') ? 'color' : 'text';
```

If `prop` can be null the first time around, I don't see why it couldn't be null the second.